<launch>
    <arg name="robot_name"                        default="picar"/>
    <arg name="threshold"                         default="0.75"/>
    <arg name="controller"                        default="ps4"/>
    <node
        pkg="joy"
        type="joy_node"
        name="ps3_joy"
        output="screen"
        ns="$(arg robot_name)">
        <param name="dev"                             value="/dev/input/js0"/>
        <remap from="joy"                             to="picar_commands/joy_raw"/>
    </node>

    <node
        pkg="picar_joy_stick"
        type="picar_joy.py"
        name="picar_joy"
        output="screen"
        ns="$(arg robot_name)">
        <param name="controller" value="$(arg controller)"/>
        <param name="threshold" value="$(arg threshold)"/>
    </node>

    <node
        pkg="picar_joy_stick"
        type="picar_robot.py"
        name="picar_robot"
        output="screen"
        ns="$(arg robot_name)">
    </node>

</launch>